src/test/java/org/folio/des/config/FolioExecutionContextHelperTest.java

```diff
@@ -0,0 +1,84 @@
+package org.folio.des.config;
+
+import org.folio.des.support.BaseTest;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.integration.XOkapiHeaders;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+
+public class FolioExecutionContextHelperTest extends BaseTest {
+
+  @Autowired
+  private FolioExecutionContextHelper contextHelper;
+
+  private static final String SYSTEM_USER = """
+    {
+      "users": [
+        {
+          "username": "data-export-system-user",
+          "id": "a85c45b7-d427-4122-8532-5570219c5e59",
+          "active": true,
+          "departments": [],
+          "proxyFor": [],
+          "personal": {
+            "addresses": []
+          },
+          "createdDate": "2021-03-17T15:30:07.106+00:00",
+          "updatedDate": "2021-03-17T15:30:07.106+00:00",
+          "metadata": {
+            "createdDate": "2021-03-17T15:21:26.064+00:00",
+            "updatedDate": "2021-03-17T15:30:07.043+00:00"
+          }
+        }
+      ],
+      "totalRecords": 1,
+      "resultInfo": {
+        "totalRecords": 1,
+        "facets": [],
+        "diagnostics": []
+      }
+    }
+    """;
+
+  @Test
+  void shouldGetFolioExecutionContextHelper() {
```

Comment:
maybe just shouldGetFolioExecutionContext?

Reply:
Thank you, corrected